iceoryx_posh/test/moduletests/test_roudi_memory_provider.cpp

```diff
@@ -38,10 +40,17 @@ class MemoryProviderFailingCreation : public iox::roudi::MemoryProvider
     {
         return iox::cxx::error<MemoryProviderError>(MemoryProviderError::MEMORY_DESTRUCTION_FAILED);
     }
+
+    static const char* getErrorStringMemoryProvider(const MemoryProviderError error)
```

Comment:
Considering the redirection internally, is this function really useful? Looks like it could be removed and replaced by the call directly.

Reply:
OK. As commented elsewhere, it is due to protected access.